lsql/judge/templates/hint_table.html

```diff
@@ -0,0 +1,30 @@
+{% extends "base_menu.html" %}
+{% load i18n %}
+{% load static %}
+{% block contenido %}
+<h1>{% translate "Página de pistas de " %}<span class="headAchievement">{{ user }}</span></h1>
+{% if used %}
```

Comment:
Para qué sirve este condicional? Puedes hacer el bucle for siempre: si elems es vacío no mostrará nada. Así que `used` sobraría